### PR TITLE
Unify API

### PR DESCRIPTION
**Pull request by sleipnir**
Rename method serve to start to match with other support languages

### CODE REVIEW

File: cloudstate/cloud_state.py [hunk 1]
```diff
@@ -22,12 +22,12 @@ def register_event_sourced_entity(self, entity: EventSourcedEntity):
...
-    def serve(self):
+    def start(self):
         server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
         add_EntityDiscoveryServicer_to_server(CloudStateEntityDiscoveryServicer(self.event_sourced_entities), server)
         add_EventSourcedServicer_to_server(CloudStateEventSourcedServicer(self.event_sourced_entities),server)
-        port = os.environ.get('HOST', '127.0.0.1') + ':' + os.environ.get('PORT', '50051')
+        port = os.environ.get('HOST', '127.0.0.1') + ':' + os.environ.get('PORT', '8080')
         server.add_insecure_port(port)
-        pprint('Starting CloudStateEntityDiscoveryServicer on ' + port)
+        pprint('Starting CloudState on ' + port)
```
Comment (marcellanz) [inline cloudstate/cloud_state.py]: wording is Cloudstate, should we stick with that? I know we use **CloudState**EntityDiscoveryServicer, also in other repos, but we start "Cloudstate" here. wdyt @sleipnir

Comment (sleipnir) [reply cloudstate/cloud_state.py]: I don't think we upload a service individually but the Cloudstate protocol itself. In other languages we see more logs of various parts of what is happening in the start phase but in the end what we are offering is the cloudstate itself

Comment (marcellanz) [reply cloudstate/cloud_state.py]: oh I just wrote that because of upper lower case wording of Cloudstate, vs CloudState

Comment (sleipnir) [reply cloudstate/cloud_state.py]: I understood that it was another question

Comment (sleipnir) [reply cloudstate/cloud_state.py]: I think the good thing is how you like it best! I always write Cloudstate

Comment (sleipnir) [reply cloudstate/cloud_state.py]: Like a God ;)

Comment (marcellanz) [reply cloudstate/cloud_state.py]: I just strive for consistency... 😅

Comment (sleipnir) [reply cloudstate/cloud_state.py]: Is that I understand it as a "Noun / Proper Name" and that's why I always use the first capital letter, but tell me how you think it should be and I make the adjustment

Comment (marcellanz) [reply cloudstate/cloud_state.py]: Lightbend went from initially CloudState to Cloudstate to name it... that's the only reasoning…

Comment (marcellanz) [reply cloudstate/cloud_state.py]: so I think we should write
`pprint('Starting Cloudstate on ' + port)`

Comment (sleipnir) [reply cloudstate/cloud_state.py]: Just now I saw the capital letter 'S'

Comment (sleipnir) [reply cloudstate/cloud_state.py]: mad max

Comment (sleipnir) [reply cloudstate/cloud_state.py]: Fixed

Comment (marcellanz) [reply cloudstate/cloud_state.py]: Awesome! Thanks @sleipnir. I appreciate any effort for perfection; always.

**Pull request merged**